setup.py

```diff
@@ -6,7 +6,7 @@
 EXTRAS_REQUIRE = {
     "django": DJANGO_REQUIRES,
     "tests": ["pytest"] + DJANGO_REQUIRES,
-    "lint": ["flake8==3.7.9", "flake8-bugbear==19.8.0", "pre-commit~=1.20"],
```

Comment:
I prefer to keep flake8 and flake8 plugins installed in the virtualenv I'm using to develop so that my editor can access them.

Reply:
I can add flake8 back